### ISSUE TITLE
Contao 5.2.10 Gallerycreater 2.0.0-rc8 Error Backend no template

### ISSUE THREAD

**Issue by pbd-kn** [markocupic/gallery-creator-bundle#35]
Hi
Ich habe in meinem Artikel das gallerycreator element eingefügt
![grafik](https://github.com/markocupic/gallery-creator-bundle/assets/75879364/27bf43c1-90ed-404a-804c-02a95bd0cc5c)

Beim Bearbeiten des Element bekomme ich den Fehler, dass das Template zum bearbeiten nicht vorhanden ist.
![grafik](https://github.com/markocupic/gallery-creator-bundle/assets/75879364/0d0e7fd3-0338-4c40-b7e8-dca8cf9be81d)

Was kann ich tun.
LG
Peter

**Comment by cgoIT** [comment]
same problem here :(

**Comment by markocupic** [comment]
@cgoIT No, this is not the correct way you should implement custom templates. Instead of this you should store your customized template under `contao/templates/_new/content_element/gallery_creator/custom.html.twig`.  Under `contao/templates/_new` you should further more create the empty dot file `.twig-root` to tell Symfony to handle the directory as a Twig root.

<img width="287" alt="image" src="https://github.com/markocupic/gallery-creator-bundle/assets/1525166/1c008e11-791b-4455-918e-344fb1ba1403">

**Comment by cgoIT** [comment]
Hi,

if so I think you missed to add some template in 2.0.0-RC7 and RC8. :(

**Comment by markocupic** [comment]
Try the [contao5 ](https://github.com/markocupic/gallery-creator-bundle/tree/contao5) branch

**Comment by cgoIT** [comment]
dev-contao5 version is working as expected in my C5 installation.
